test/support/factory.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Livebook.Factory do
+  use ExMachina
```

Comment:
Do you think we need ExMachina?

I think we can implement `build` as:

```elixir
def build(factory_name, attrs) do
  struct = apply(__MODULE__, factory_name, [])
  struct!(struct, attrs)
end
```

There is also this guide: https://hexdocs.pm/ecto/test-factories.html

Reply:
This would work too, TIL